.github/workflows/schema.yml

```diff
@@ -0,0 +1,44 @@
+name: Update schema
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1-5" # Weekdays
+
+jobs:
+  update:
+    if: github.repository == "aws/serverless-application-model"
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
```

Comment:
What does this `contents` permission do?

Reply:
Allows to push commits: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Got it 👍